frontend/start_docker.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/sh
+echo "Starting the container for the angular website!\n"
+read -p "Do you want to build a new image? (This process could take a while) [Y/N]: " input
+
+if [ $input = 'Y' ]; then
+    docker build -t angular-website:latest .
+fi
+docker run -p 4200:4200 --name angular-website angular-website:latest
```

Comment:
should the angular port not be 443?

Reply:
The angular port currently runs on default value. That value is port 4200. Since the team didn't agree on a specific port yet, we should keep the default value untill that discussion is held.

Reply:
yes